src/pyfmi/fmil_import.pxd

```diff
@@ -467,64 +467,83 @@ cdef extern from 'fmilib.h':
         pass
 
     #FMI SPECIFICATION METHODS (2.0)
+    # basic
```

Comment:
Don't bother reviewing this; what has been done:

1. Removed a few commented-out functions
2. Sorted functions into CPAI vs non CAPI. There was some existing sorting, but neither complete nor correct.